docs/current_docs/api/module-structure.mdx

```diff
@@ -165,12 +168,35 @@ py-modules = ["main"]
 </TabItem>
 </Tabs>
 
-:::note
-Build backends are independent from installers (or *build frontends*, see [PEP 517](https://www.python.org/dev/peps/pep-0517/)), even though some installers may provide both (as separate packages). For example, [hatch](https://pypi.org/project/hatch/) vs [hatchling](https://pypi.org/project/hatchling/), [poetry](https://pypi.org/project/poetry/) vs [poetry-core](https://pypi.org/project/poetry-core/).
 :::
 
-:::tip
-`dagger init` won't override existing `pyproject.toml` and `.py` files, so it's possible to use an external process to generate a different template, before calling `dagger init`.
+:::tip CUSTOMIZE IMPORT PACKAGE
+The Python SDK looks for the main object of the Dagger module in the Pyhton [import package that is named after the distribution package name](https://packaging.python.org/en/latest/discussions/distribution-package-vs-import-package/#how-do-distribution-package-names-and-import-package-names-compare) (in particular, using underscores `_` as a word separator).
```

Comment:
```suggestion
The Python SDK looks for the *main object* of the Dagger module in the Python [import package that is named after the distribution package name](https://packaging.python.org/en/latest/discussions/distribution-package-vs-import-package/#how-do-distribution-package-names-and-import-package-names-compare) (in particular, using underscores `_` as a word separator).
```

Formatting this for consistency with other similar usage in this doc (although in general I always prefer not to use highlights in the absence of agreed conventions as it can create mixed signals for readers)